src/app/components/BodyHandsPage.tsx

```
@@ -592,26 +616,31 @@ function ProductCard({
           {/* Minimal Add to Cart Button - Aesop Style */}
           <div className="flex items-center justify-end">
             <button
-              onClick={() => handleAddToCart(product, selectedSize, getCurrentPrice())}
+              onClick={() => !isInCart && handleAddToCart(product, selectedSize, getCurrentPrice())}
               onMouseEnter={() => setIsButtonHovered(true)}
               onMouseLeave={() => setIsButtonHovered(false)}
-              className="group/btn relative inline-flex items-center gap-2 pb-0.5"
+              disabled={isInCart}
+              className={`group/btn relative inline-flex items-center gap-2 pb-0.5 ${isInCart ? "cursor-default opacity-60" : "cursor-pointer"}`}
             >
               <span
```

Biome (2.3.13)

[error] Provide an explicit type prop for the button element.

The default type of a button is submit, which causes the submission of a form when placed inside a `form` element. This is likely not the behaviour that you want inside a React application.
Allowed button types are: submit, button or reset

(lint/a11y/useButtonType)